docs/assessment.md

```diff
@@ -214,10 +216,12 @@ The exact details of how PrairieLearn awards points are as follows. First, a num
 - **Question value** is the current maximum number of points that the question is worth. This is equal to `autoPoints[n]` for the `n`-th attempt at the question.
 - **Question points** is the total number of points earned by the student for the question. This will never decrease. This is initialized to zero.
 
-For the `n`-th submission attempt made by a student, the scoring algorithm is as follows:
+When a student submits an answer to a question, the submission is first checked for validity. For example, the `pl-number-input` element considers a non-numeric input to be invalid. If the submission is valid then it is next determined whether it should be graded. A submission is only graded if explicitly requested by the student using the "Save & Grade" button, or if was the last submission when the assessment was closed.
```

Comment:
Same as above.

Reply:
Changed in 73bed39950cc838d14c92387045d200036099085